opentelemetry-proto/src/proto.rs

```diff
@@ -295,7 +302,7 @@ pub mod tonic {
     #[path = ""]
     pub mod profiles {
         #[path = "opentelemetry.proto.profiles.v1development.rs"]
-        pub mod v1;
+        pub mod v1development;
```

Comment:
Note for reviewers: it renames the package which is a breaking change, but the package should be `v1development` and it is unstable and breaking changes are expected in the `v1development`

Reply:
cc @cijothomas

Reply:
Can we mention this specifically as breaking change in changelog.

Reply:
seems reasonable to me apologies for not doing it this way in the initial pr

Reply:
Thanks. Applied a note on breaking change in the changelog.